config/query.config.prod.js

```diff
@@ -0,0 +1,8 @@
+const db = require('../config/db.config.prod');
+
+const query = (sqlQuery, cb) => {
+    // filter out reserver by default because we want tickets that have not been taken yet
+    db.query(sqlQuery,cb);
+};
+
+module.exports.query = query;
```

Comment:
Why not inline that into here? (I.e. something like the following)

```javascript
module.exports.query = (sqlQuery, cb) => {
        db.query(sqlQuery, cb);
}
```

(Or, if it'll work, better yet just `module.exports.query = db.query`.)